.changeset/yellow-schools-matter.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core': patch
+---
```

Comment:
Hi! Could you make this change in the new core-components module instead? @Rugvip or both if we aren't pulling the plug on the old core quite yet?

Reply:
I copied the updated file to the `@backstage/core-components` package.